### ISSUE TITLE
Converting station level stationxml a Exception occurs

### ISSUE THREAD

**Issue by jholland-usgs** [iris-edu/stationxml-seed-converter#52]
Sample xml attached as .txt.
[stationlevel.txt](https://github.com/iris-edu/stationxml-seed-converter/files/4589948/stationlevel.txt)

Conversion works as expected with version 2.0.10, but fails with later version of the converter.

Error generated:
```
 java -jar ~/Downloads/stationxml-seed-converter-2.1.0-RC-2.jar --input ~/tmp/validationStage2PZ/stationlevel.xml --output out.seed
[2020-05-06 17:25:53] [SEVERE] edu.iris.dmc.station.Application run: edu.iris.dmc.station.Application.exitWithError(Application.java:191)
edu.iris.dmc.station.Application.convert(Application.java:181)
edu.iris.dmc.station.Application.run(Application.java:123)
edu.iris.dmc.station.Application.main(Application.java:68)
```

**Comment by timronan** [comment]
I just ran the same test on your file and received:
```
java -jar ./stationxml-seed-converter-2.0.1-SNAPSHOT.jar --input ./stationlevel.xml --output ./stationlevel.xml.dataless
java.io.IOException: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory]
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:47)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:29)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:19)
	at edu.iris.dmc.station.Application.convert(Application.java:106)
	at edu.iris.dmc.station.Application.run(Application.java:62)
	at edu.iris.dmc.station.Application.main(Application.java:18)
Caused by: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory]
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:269)
	at javax.xml.bind.ContextFinder.find(ContextFinder.java:412)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:721)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:662)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.load(XmlToSeedFileConverter.java:65)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.load(XmlToSeedFileConverter.java:60)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:40)
	... 5 more
Caused by: java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at javax.xml.bind.ServiceLoaderUtil.nullSafeLoadClass(ServiceLoaderUtil.java:122)
	at javax.xml.bind.ServiceLoaderUtil.safeLoadClass(ServiceLoaderUtil.java:155)
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:267)
	... 11 more

Error: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory]


Usage:
java -jar stationxml-converter.jar [arguments]
	-h, aliases = "--help", usage = "print this message"
	--input, usage = "Input as a file or URL"
	--output, usage = "Output file path and name"
(base) Tims-1171:Downloads tronan$ vim ./stationlevel.xml
(base) Tims-1171:Downloads tronan$ java -jar ./stationxml-seed-converter-2.0.1-SNAPSHOT.jar --input ./stationlevel.xml --output ./stationlevel.xml.dataless
java.io.IOException: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory]
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:47)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:29)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:19)
	at edu.iris.dmc.station.Application.convert(Application.java:106)
	at edu.iris.dmc.station.Application.run(Application.java:62)
	at edu.iris.dmc.station.Application.main(Application.java:18)
Caused by: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory]
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:269)
	at javax.xml.bind.ContextFinder.find(ContextFinder.java:412)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:721)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:662)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.load(XmlToSeedFileConverter.java:65)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.load(XmlToSeedFileConverter.java:60)
	at edu.iris.dmc.station.converter.XmlToSeedFileConverter.convert(XmlToSeedFileConverter.java:40)
	... 5 more
Caused by: java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at javax.xml.bind.ServiceLoaderUtil.nullSafeLoadClass(ServiceLoaderUtil.java:122)
	at javax.xml.bind.ServiceLoaderUtil.safeLoadClass(ServiceLoaderUtil.java:155)
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:267)
	... 11 more

Error: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.bind.v2.ContextFactory]
```
We do not support the conversion of that is truncated above the response level. The converter may convert channel level metadata but the output are not supported.

**Comment by timronan** [comment]
This issue has been resolved by commit 538408ff7e6055c5caf193df9a50c6c1e29530ae. Please build the project from source and verify.

**Comment by jholland-usgs** [comment]
Confirmed this now doesn't crash anymore. Thank you.
